jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/ToolkitAuthManager.kt

```diff
@@ -143,6 +144,18 @@ fun loginSso(project: Project?, startUrl: String, scopes: List<String> = ALL_SON
     }
 }
 
+fun logoutFromSsoConnection(project: Project?, connection: ToolkitConnection, callback: () -> Unit = {}) {
+    if (connection !is AwsBearerTokenConnection) return
```

Comment:
I think it makes more sense to type `connection` as `AwsBearerTokenConnection` in the signature instead of returning early. Especially because the function is called `logoutFromSsoConnection` instead of `logoutFromConnection`